test/api-callbacks.js

```diff
@@ -39,18 +39,12 @@ var auctionExample = require('./fixture/auctionexample1');
 
 describe('Api Callbacks - Tests', function() {
 
-  function clearEvents() {
```

Comment:
Simply use the new `PubfoodEvent.removeAllListeners()` method.

This cleanup is repeated for a couple of test files.